### ISSUE TITLE
Document supported Python versions

### ISSUE THREAD

**Issue by bmw** [letsencrypt/letsencrypt#1589]
People have tried running Python on older OSs and have seen:
```
You have an ancient version of Python entombed in your operating system...
```
We should document what versions of Python we support in both the README and readthedocs.

**Comment by cowlicks** [comment]
Anyone know what 3.x versions are supported? 3.3+?

**Comment by bmw** [comment]
@cowlicks, the client does not support Python 3.x at this time. We plan to add Python 3.x support after we launch.

Closed by #1593.
